<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-OJ618SO" ignored="0" name="Default test" tests="2" failures="1" timestamp="26 Feb 2021 06:09:04 GMT" time="40.65" errors="0">
  <testcase name="runScenario" time="30.774" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Parallel.CompanyTest.user_delete_the_company_from_subsource(CompanyTest.java:86)
at âœ½.user delete the company from subsource(file:///D:/SubSourceV2/SubSource/CodeBase/./src/test/resources/Parallel/Company.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="35.627" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
</testsuite> <!-- Default test -->
